libs/ledger/src/storage_unit/storage_unit_client.cpp

```diff
@@ -183,7 +183,7 @@ bool StorageUnitClient::RevertToHash(Hash const &hash, uint64_t index)
   {
     assert(!hash.empty());
 
-    FETCH_LOG_INFO(LOGGING_NAME, "reverting tree leaf: ", lane_merkle_hash.ToHex());
+    FETCH_LOG_DEBUG(LOGGING_NAME, "reverting tree leaf: ", lane_merkle_hash.ToHex());
```

Comment:
I would leaf this as an info as it only happens on a fork/revert right, so not that common a code path in our system